runWPT.sh

```diff
@@ -8,40 +8,51 @@ log() {
 }
 
 usage() {
-  log "Usage: [HEADLESS=<true | false>] [UPDATE_EXPECTATIONS=<true | false>] $0 [webdriver/tests/bidi/[...]]"
+  log "Usage: [CHROMEDRIVER=<true | false>] [HEADLESS=<true | false>] [UPDATE_EXPECTATIONS=<true | false>] $0 [webdriver/tests/bidi/[...]]"
 }
 
 if [[ $# -gt 0 && ("$1" == "-h" || "$1" == "--help") ]]; then
   usage
   exit 0
 fi
 
+# The path to the browser binary.
+readonly CHROME_BIN="${CHROME_BIN:-/Applications/Google Chrome Dev.app/Contents/MacOS/Google Chrome Dev}"
+
+# Whether to use Chromedriver with mapper.
+readonly CHROMEDRIVER="${CHROMEDRIVER:-false}"
```

Comment:
Defining this variable means we can easily run `runWPT.sh` locally as it's done in CI (for the chromedriver setting).